qumcmc/quantum_mcmc_routines.py

```diff
@@ -225,3 +225,41 @@ def quantum_enhanced_mcmc(
             pass
 
     return mcmc_chain
+
+####
+## decorator class for running quantum_enhanced_mcmc() 
+from dataclasses import dataclass
+from .mcmc_sampler_base import mcmc_sampler
+@dataclass
```

Comment:
> I am not convinced these things should be data classes. They don't move around data. They are proper classes with functionality.

Sure 

> The run can be renamed to train where the training options are passed to, instead of the init of the class. As it is now, the class is a very fine wrapper around the training function 

Like I alrady made a class for training in the `training` module, If I'm not wrong ou are asking to make a different training class that is more like the wrapper fro smapling I made ?

Reply:
Data classes should not have methods that actually run complicated code. I would say to either not make a training class, or commit and make a `BoltzmannMachine` class that wraps everything.